sql/1709716115361-add-sort-column.sql

```diff
@@ -0,0 +1,4 @@
+CREATE SEQUENCE periodtracker.article_sorting_key INCREMENT 1 MINVALUE 1 MAXVALUE 9223372036854775806 START 1 CACHE 1;
```

Comment:
CREATE SEQUENCE Statement:
The CREATE SEQUENCE statement creates a sequence named article_sorting_key with specific parameters. It generates unique values for the sortingKey column.
Ensure that the sequence name and parameters are appropriate for your use case. The INCREMENT, MINVALUE, MAXVALUE, START, and CACHE options define how the sequence generates values.
Ensure that the sequence name periodtracker.article_sorting_key is valid and reflects your schema structure.

Reply:
-- Create a sequence to generate sorting keys
CREATE SEQUENCE periodtracker.article_sorting_key
INCREMENT 1
MINVALUE 1
MAXVALUE 9223372036854775806
START 1
CACHE 1;